dev/MRTCore/mrt/Core/unittests/UnitTest.vcxproj

```diff
@@ -135,6 +135,12 @@
       <SubSystem>Windows</SubSystem>
       <AdditionalLibraryDirectories>$(VCInstallDir)UnitTest\lib;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
     </Link>
+    <PostBuildEvent>
```

Comment:
nit: is it possible to do it once, instead for each flavor?

Reply:
I tried making it its own property group, but that does not work.

We can coalesce the entire section into one. Most entries are identical, so we could only fork the few that are different. I considered doing that. The problem is that you're fighting against VS when you do so. Once you make changes in the GUI it will once again break it out. We are probably not going to rev this a lot so we can consider.